webapp/utils/utils.jsx

```diff
@@ -737,19 +737,11 @@ export function resetTheme() {
     applyTheme(Constants.THEMES.default);
 }
 
-export function applyFont(fontName) {
+export function applyOpenSansFont() {
```

Comment:
I don't think we need this, we can just add that class to the body tag unless I'm mistaken